person.rb

```diff
@@ -2,7 +2,7 @@
 
 # main class Person
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Let's implement `add_rental` method in the class Person that accepts book and date as parameters and creates a new rental for the person. Cheers :+1:

Reply:
@sumairq Thanks, I will do that, but can you tell me, where is that in the requirements?

Reply:
Hi @MattGomb  As we know that a person has many rentals and a rental belongs to a single person. Naturally we must have a method adding the rental for the person.

It should look like this:

```  
 def add_rental(book, date)
 
   Rental.new(date, book, self)
 
 end
`

Reply:
@sumairq okay, thanks! :)